hypertrace-view-generator/hypertrace-view-generator/src/main/java/org/hypertrace/viewgenerator/generators/SpanEventViewGenerator.java

```diff
@@ -290,7 +290,7 @@ String getRequestUrl(Event event, Protocol protocol) {
     switch (protocol) {
       case PROTOCOL_HTTP:
       case PROTOCOL_HTTPS:
-        return EnrichedSpanUtils.getFullHttpUrl(event).orElse(null);
+        return EnrichedSpanUtils.getFullHttpUrl(event).orElse(event.getHttp().getRequest().getPath());
```

Comment:
Could we elaborate on why this value and not null? @buchi-busireddy @kotharironak

Reply:
While showing URL in the table view of spans or API traces, it makes sense to show the URL path if full URL is missing. Currently we see it empty for some spans.

Reply:
No, no disagreement. I was more asking on such argument as a comment in the code so in the future someone does not decide `null` is a better value.

Reply:
The part after this, makes it all more curious. I would advise to have coherent and fixed concepts except in span name, vs mixes ex full url or just the path part, or whatever the below returns. one can argue "requestUrl" is talking about url and a path it technically a url, but at the same time it seems this is a muddy result which can break aggregation and analysis.

```
      case PROTOCOL_GRPC:
        return event.getEventName();
```